src/stories/page.css

```diff
@@ -1,5 +1,9 @@
+.css-rc1jbh {
+  background: linear-gradient(180deg, #669df6 0%, #669df6 65.89%, #4ecde6 115%);
```

Comment:
We should define the to and from colors as DaisyUI variables and then use TailwindCSS utilities to make the gradient.